src/njoy21/input/COVR/Card4/Mat.hpp

```diff
@@ -0,0 +1,12 @@
+struct Mat : argument::common::Matd {
```

Comment:
Where is this tested?

Reply:
[Here](https://github.com/njoy/NJOY21/blob/feature/COVR/src/njoy21/input/COVR/Card4/test/Card4.test.cpp).